tests/make.zunit

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env zunit
+
+@setup {
+    load ../fzf-zsh-completions.plugin.zsh
+    TEMP=$(mktemp -d)
```

Comment:
My bad... I don't think we need TEMP for this case. Just `pushd` to `tests/_support/make`.

Reply:
Ok.